schematron/Consolidation.sch

```diff
@@ -1277,8 +1277,8 @@ Schematron generated from Trifolia on 7/20/2021
       <sch:assert id="a-1098-7499" test="count(cda:templateId[@root='2.16.840.1.113883.10.20.22.4.16'][@extension='2014-06-09'])=1">SHALL contain exactly one [1..1] templateId (CONF:1098-7499) such that it SHALL contain exactly one [1..1] @root="2.16.840.1.113883.10.20.22.4.16" (CONF:1098-10504). SHALL contain exactly one [1..1] @extension="2014-06-09" (CONF:1098-32498).</sch:assert>
       <sch:assert id="a-1098-7500" test="count(cda:id) &gt; 0">SHALL contain at least one [1..*] id (CONF:1098-7500).</sch:assert>
       <sch:assert id="a-1098-7507" test="count(cda:statusCode)=1">SHALL contain exactly one [1..1] statusCode (CONF:1098-7507).</sch:assert>
-      <sch:assert id="a-1098-7508-c" test="cda:effectiveTime[@xsi:type='IVL_TS']">SHALL contain exactly one [1..1] effectiveTime (CONF:1098-7508) such that it</sch:assert>
-      <sch:assert id="a-1098-28499-c" test="not(tested-here)">**SHALL** contain exactly one [1..1] @xsi:type="PIVL_TS" or "EIVL_TS" (CONF:1098-28499).</sch:assert>
+      <sch:assert id="a-1098-7508-c" test="count(cda:effectiveTime[(cda:low or @value) and not(cda:low and @value)])=1">SHALL contain exactly one [1..1] effectiveTime (CONF:1098-7508) such that it SHALL contain either a low or a @value but not both (CONF:1098-32890).</sch:assert>
+      <sch:assert id="a-1098-28499-c" test="count(cda:effectiveTime[@operator='A'])=0 or count(cda:effectiveTime[@operator='A' and (@xsi:type='PIVL_TS' or @xsi:type='EIVL_TS')])=1">**SHALL** contain exactly one [1..1] @xsi:type="PIVL_TS" or "EIVL_TS" (CONF:1098-28499).</sch:assert>
```

Comment:
I think there are multiple interpretations here, so I'm firing off a question to the co-chairs / maybe will make it a committee topic if we can't agree.

Possible interpretations:
- The whole thing is a SHOULD, so even if they have multiples, the worst they should get is a warning
- The SHOULD has max cardinality=1, so putting more than 1 is clearly an error
- In either case - does the warning/error fire when you have more than 1 operator=A or more than one operator=A WITH xsi:type=EIVL/PIVL?

For that 3rd bullet, I interpreted it as only operator=A because that's the only statement marked as a branch identifier in Trifolia (the xsi:type check was just because I couldn't do that in its own conformance statement because Trifolia nested it in a "warning" branch). However - Trifolia won't allow you to mark primitive constraints as branch identifiers and both SHALL's are at the same level, so it makes some sense to consider them as both identifiers (i.e. you could theoretically have other effectiveTime with operator=A as long as their type was SXCM_TS, IVL_TS or SXPR_TS and they didn't contain `@value` or `low`).